druid/src/widget/mod.rs

```diff
@@ -52,8 +50,6 @@ mod view_switcher;
 mod widget;
 mod widget_ext;
 
-#[cfg(feature = "image")]
-#[cfg_attr(docsrs, doc(cfg(feature = "image")))]
 pub use self::image::{Image, ImageData};
```

Comment:
I don't think this `self` is needed in front of this import. Can you check if it compiles without it?